web/profiles/custom/os2loop/modules/os2loop_question/assets/js/hideReferenceIds.js

```diff
@@ -9,33 +8,32 @@
     Drupal.behaviors.autocompleteReferenceEntityId = {
       attach: function (context) {
         // Remove reference IDs for autocomplete elements on init.
-        $('.form-autocomplete', context).once('replaceReferenceIdOnInit').each(function () {
+        $(once('replaceReferenceIdOnInit', '.form-autocomplete', context)).each(function () {
```

Comment:
This looks weird. Is the `once` function suddenly defined in the global scope?

Reply:
Added core/once as a library dependency and in wrapper

Reply:
https://www.drupal.org/node/3158256

Reply:
I'm pretty I don't understand what `$(once(…))` actually does, but I trust you have tested it and that it works.